src/test/acceptance/features/navigation.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Application process navigation is controlled
+  In order to facilitate the beta HTBHF programme
```

Comment:
How about 'In order to help me complete the application correctly'?

Reply:
Yep, done.